parsl/executors/high_throughput/executor.py

```diff
@@ -551,6 +551,7 @@ def _start_local_interchange_process(self) -> None:
         logger.debug("Popened interchange process. Writing config object")
         stdin.write(config_pickle)
         stdin.flush()
+        stdin.close()
         logger.debug("Sent config object. Requesting worker ports")
```

Comment:
Suggest adding some version of a UT for this.  It's not an important detail, per se, but it's also a detail that I can see being easily forgotten/missed in a refactor.  Especially from newer contributors.

Reply:
#3397 is a longer term plan to catch all leftover fds and threads